config/environments/production.rb

```diff
@@ -60,8 +60,12 @@
   # the I18n.default_locale when a translation can not be found)
   config.i18n.fallbacks = [I18n.default_locale]
 
-  # Send deprecation notices to registered listeners
-  config.active_support.deprecation = :notify
+  # Send deprecation notices to registered listeners.
+  # config.active_support.deprecation = :notify
+
+  # Change from rails :notify default since it's unlikely we'll have users setup
+  # notifications for deprecation warnings.
+  config.active_support.deprecation = :log
```

Comment:
This is the 🍖 of the PR.  The default of notify is great when hosted as you can setup notifications for these.  Since most of our users are not hosted by us, it will be unlikely we'll ever get deprecations reported to us unless we get them by default from logging.

Reply:
I'm not sure...see my comment below.  What does this solve that we couldn't do previously?

Reply:
Deprecations, especially rails ones, should be exceptional.  I'd like to stay on top of them so future upgrades are easier.  Note, this doesn't affect `Vmdb::Deprecation` as that will never log in production so this only logs rails deprecations, ones we should definitely stay on top of.

Reply:
Don't we have a lot of code suppressing the notifications in production? At least for our own deprecations.